std/math.d

```diff
@@ -6832,6 +6832,141 @@ if (isFloatingPoint!(F) && isFloatingPoint!(G))
     assert(approxEqual(pow(twoI, three), 8.0));
 }
 
+/** Computes the value of a positive integer x, raised to the power n, modulo m.
+ *
+ * All values have to be unsigned types, otherwise it will raise a compilation error.
+ */
+Unqual!(Largest!(F, H)) powmod(F, G, H)(F x, G n, H m)
+if (isIntegral!F && isIntegral!G && isIntegral!H &&
```

Comment:
You don't need `isIntegral`, as `isUnsigned` already allow only a subset of the types for which `isIntegral` is true.